src/Console/Command/DescribeCommand.php

```diff
@@ -52,9 +52,6 @@
 #[AsCommand(name: 'describe')]
 final class DescribeCommand extends Command
 {
-    /**
-     * @var string
```

Comment:
why type removal?

Reply:
phpDoc on const is useless - type is clear by looking at value, can be inferred automatically, and does not prevent type overriding in child classes, which can re-define consts.

Reply:
> phpDoc on const is useless

but this is regular `property`, not `constant`

> and does not prevent type overriding in child classes

that's the thing, we don't want type to be changed in child class

Reply:
I replied without looking at the diff, and it seems like I have short memory 😅. It's because parent's type (`Symfony\Component\Console\Command\Command`) is `string|null`, so PHPStan with strict types [reports an error](https://phpstan.org/r/112ba957-0598-42ae-8aed-2947c1ee0a9e). [Native type can't be added](https://phpstan.org/r/7036f4ff-3e8e-4be5-97fc-5a8c71fb59ba), and re-declaring the same type is not needed as PHPStan [infers it from the parent](https://phpstan.org/r/6db3e2d9-8a60-4d53-89b4-8954d21c9b5f). So it was useless.